pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -94,16 +94,14 @@ in stdenv.mkDerivation {
       --replace /usr/share/ $out/share/
   '' + stdenv.lib.optionalString (theme != null) ''
     cat <<EOF >> $out/lib/slack/resources/app.asar.unpacked/src/static/ssb-interop.js
+    var fs = require('fs');
     document.addEventListener('DOMContentLoaded', function() {
     let tt__customCss = ".menu ul li a:not(.inline_menu_link) {color: #fff !important;}"
-    $.ajax({
-        url: '${theme}/theme.css',
-        success: function(css) {
+      fs.readFile('${theme}/theme.css', 'utf8', function(err, css) {
             \$("<style></style>").appendTo('head').html(css + tt__customCss);
```

Comment:
Make sure to properly re-indent :slightly_smiling_face:

Reply:
it is properly indented, original one was not. I left it like that to minimize diff surface to avoid any future conflicts (just to get this in faster)

Reply:
Ok, my bad!